src/components/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from '../styles/loading'
+
+import React from 'react'
+
+const Loading = ({ message }) => {
+  return (
+    <div className={styles['fil-loading']}>
+      <p>{message}</p>
+    </div>
+  )
+}
+
+export default Loading
```

Comment:
@enguerran like a lot of things in life, it depends: with some simple components like `Loading`, it makes totally sense to me to pass a translated string as arg. For more complex (but still dumb) components, it'll sometimes (often) be necessary to translate things inside the component. I'm against any hard rules related to this concern, let's simply use our best judgement when writing components.